exercises/practice/after-party/.docs/instructions.md

```diff
@@ -0,0 +1,50 @@
+# After Party
+
+We're back to playing the role of a bouncer.
```

Comment:
Alas, while I was hoping we could reuse instructions verbatim, this makes a reference to the Guest List exercise.

We can't assume students completed that exercise already so this sentence needs to be modified.  Maybe something like this.

```suggestion
We're back to playing the role of a bouncer at the Oscars, and need a function to check if someone is on the Guest List.
```